### PR TITLE
Update double-key-encryption.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the version of .net core as 3.1 is deprecated.

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10754

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/double-key-encryption.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ When you're done, you can encrypt documents and files using DKE. For information
 
 Install these prerequisites on the computer where you want to install the DKE service.
 
-**.NET Core 3.1 SDK**. Download and install the SDK from [Download .NET Core 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1).
+**.NET Core 7.0 SDK**. Download and install the SDK from [Download .NET Core 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
**.NET Core 7.0 SDK**. Download and install the SDK from [Download .NET Core 7.0](https://dotnet.microsoft.com/download/dotnet/7.0).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**